cv_scan_ai/requirements.txt

```diff
@@ -0,0 +1,30 @@
+aiohttp==3.8.5
+aiosignal==1.3.1
+async-timeout==4.0.2
+attrs==23.1.0
+certifi==2023.7.22
+charset-normalizer==3.2.0
+dataclasses-json==0.5.14
+frozenlist==1.4.0
+idna==3.4
+langchain==0.0.257
+langsmith==0.0.19
+marshmallow==3.20.1
+multidict==6.0.4
+mypy-extensions==1.0.0
+numexpr==2.8.5
+numpy==1.25.2
+openai==0.27.8
+openapi-schema-pydantic==1.2.4
+packaging==23.1
+pydantic==1.10.12
+python-dotenv==1.0.0
+PyYAML==6.0.1
+requests==2.31.0
+SQLAlchemy==2.0.19
+tenacity==8.2.2
+tqdm==4.65.1
+typing-inspect==0.9.0
+typing_extensions==4.7.1
+urllib3==1.26.6
+yarl==1.9.2
```

Comment:
do we really need all these libraries for such a basic task? if yes please explain

Reply:
it's a list of dependencies of big libraries we use, removing langchain will make it shorter.

Reply:
Ok, sounds like one more reason to use openai. Hopefully, it will make it shorter.

Reply:
its will not be shorter anyway